app/controllers/concerns/phone_confirmation_flow.rb

```diff
@@ -76,7 +76,7 @@ def send_confirmation_code
     job.perform_later(
       code: confirmation_code,
       phone: unconfirmed_phone,
-      otp_created_at: current_user.direct_otp_sent_at.to_s
+      otp_created_at: (current_user.direct_otp_sent_at || Time.zone.now).to_s
```

Comment:
I guess it begs the question: why have it as an argument at all? The job being created will always be co-incident with the OTP being created. Why not just record the current time within the job and leave it at that?